internal/configs/virtualserver.go

```diff
@@ -2179,6 +2216,24 @@ func generateErrorPageName(errPageIndex int, index int) string {
 	return fmt.Sprintf("@error_page_%v_%v", errPageIndex, index)
 }
 
+func checkGrpcErrorPageCodes(errorPages errorPageDetails, isGRPC bool, uName string, vscWarnings *Warnings) {
```

Comment:
for consistency with the other functions, let' use `vscWarnings Warnings` rather than `vscWarnings *Warnings`